.mailmap

```diff
@@ -52,9 +52,7 @@ Brigitta Sipőcz              <bsipocz@gmail.com> <bsipocz@users.noreply.github.
 Bruce Merry                  <bmerry@ska.ac.za>
 Bruce Merry                  <bmerry@ska.ac.za> <bmerry@gmail.com>
 Bryce Nordgren               <bnordgren@gmail.com>
-Chiara Marmo                 <chiara.marmo@u-psud.fr>
-Chiara Marmo                 <chiara.marmo@u-psud.fr> <cmarmo@users.noreply.github.com>
-Chiara Marmo                 <chiara.marmo@u-psud.fr> <marmochiaskl@gmail.com>
+Chiara Marmo                 <marmochiaskl@gmail.com>
```

Comment:
I don't think this is the correct edit. The purpose of the mailmap is to make all commits that have any of your email addresses show up as the same email address and name, so if you want all your commits to show with your gmail address you would need this to look like this:
```suggestion
Chiara Marmo                 <marmochiaskl@gmail.com>
Chiara Marmo                 <marmochiaskl@gmail.com> <chiara.marmo@u-psud.fr>
Chiara Marmo                 <marmochiaskl@gmail.com> <cmarmo@users.noreply.github.com>
Chiara Marmo                 <marmochiaskl@gmail.com> <marmochiaskl@gmail.com>
```

As it [says in the new docs](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#astropy-guidelines-for-git) you should run [git shortlog -es](https://git-scm.com/docs/git-shortlog) to check you only appear in the list once.